content/platform/data-services-cloud-console-api/home.md

```diff
@@ -0,0 +1,176 @@
+
+1. The list of the REST API resources that are supported as of the release. 
+2. The information about the HTTP method, parameters and the responses that are expected out for this resource
+3. The syntax for the HTTP method and path to this resource. Note that this path is relative path, for the complete path, please add the base-URL documented below to provide the full path.
+4. The body of response is returned in JSON format according to the response status of the REST API.
+
+The website also provides the links to download the DSCC open-API definitions in either json or yaml format. Below is an example of the downloaded json file from the DSCC REST API documentation website.
+
+![](/img/open-api-yaml.png)
+
+### Documentation and Tutorials
+
+The overview of the Data Services Cloud Console is available at [this link](https://www.hpe.com/us/en/storage/data-services-cloud-console.html#overview). There will be more blogs that will provide more information to help customers adopting this API with examples, code snippets and many other helpful information.
+
+### Documented Attributes
+
+* **API Name & Description** - Provides short description of the objective for this API with the supported HTTP request method (POST, GET, DELETE, PATCH, PUT etc).
+* **API Path** - Provides detail URL path as the end-point to issue the API call. Note that user must add the base path URL to extend this path to the correct resource end-point.
+* **API Parameter** - Provides the client to input information such as the object for the manipulation, select a filter to limit the returned objects, and other purposes.
+* **API Data/Body/Payload** - This the data passed along in different part of the REST API usually associated with operation such as POST/PATCH/PUT
+* **API Response** - Provides detail response information on the result of the particular API and may include more data in JSON format. 
+* **API Error Codes** - Provide result of the execution of the API, will return either good or error along with the error message due to incorrect, or unauthorized API call.
+
+### Supported API Categories (Services)
+
+The API categories for DSCC will grow in accordance to the expansion of the DSCC services that are going to released in future. Current services that are available, as recorded today:
+
+#### Common (Alletra-6K, Alletra-9K, Primera, Nimble)
+
+1. Authentication
+2. Tasks
+3. Event Audit
+4. AuthZ (User RBAC permissions)
+5. Issues
+6. controllers
+7. host-initiator-groups
+8. Host-initiators
+9. ports
+10. shelves
+11. storage-pools
+12. storage-systems
+13. system-settings
+14. volume-sets
+15. volumes
+
+#### Alletra-6K or Nimble
+
+1. protection-templates
+2. disks
+
+### Versioning
+
+The major version number will be provided in the resource path as "v1" in this example:
+
+```md
+/api/v1/<resource group>/...
+```
+
+Some examples of these resource paths that contain several resource groups under the same root:
```

Comment:
Recommend modifying this sentence to:
**Here are s**ome examples of these resource paths that contain several resource groups under the same root: